include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -347,6 +347,14 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                ///< The default allocator for execution provider
 } OrtMemType;
 
+/** \brief This mimics OrtDevice type constants so they can be returned in the API
+ */
+typedef enum OrtMemoryInfoDeviceType {
+  CPU = 0,
```

Comment:
Should this be:

OrtMemoryInfoDeviceType_CPU, etc?

Since otherwise this adds a global symbol of 'CPU' which is likely to collide somewhere.

Reply:
I agree.